web/api/swagger/mocks.yaml

```diff
@@ -723,24 +809,87 @@ definitions:
         type: "string"
         format: "date-time"
         description: "Timestamp job started"
-      output:
+      steps:
         type: "array"
-        description: "Array with Job output info (not required for UI atm - skip implementation)"
+        description: "Array of Step output info"
         items:
-          type: "string"
-      stepId:
+          type: "object"
+      user:
         type: "string"
-        description: "ID of the step being processed"
-      stepName:
+        description: "User that started the job"
+      flow:
+        type: "string"
+        description: "Name of the flow being processed"
+      lastAttemptedStep:
+        type: "integer"
+        format: "int64"
+        description: "last step that was attempted to run"
+      lastCompletedStep:
+        type: "integer"
+        format: "int64"
+        description: "last step that was completed"
+      jobStatus:
         type: "string"
-        description: "Name of the step being processed"
-      stepRunningPercent:
+        description: "Status of the job"
+        enum:
+        - "canceled"
+        - "failed"
+        - "running"
+        - "finished-with-errors"
+        - "completed"
+      successfulEvents:
+        type: "integer"
+        format: "int64"
+        description: "# of documents successfully processed"
+        minimum: 0
+      failedEvents:
         type: "integer"
         format: "int64"
-        description: "integer representing percent complete, null if not running"
+        description: "# of documents that failed to process"
         minimum: 0
-        maximum: 100
-      status:
+    example:
+      user: "admin"
+      failedEvents: 0
+      successfulEvents: 0
+      flow: "Flow 01"
+      steps: []
+      lastAttemptedStep: 0
+      lastCompletedStep: 0
+      startTime: "2000-01-23T04:56:07.000+00:00"
+      id: "id"
+      endTime: "2000-01-23T04:56:07.000+00:00"
+      jobStatus: "canceled"
+  jobs:
```

Comment:
You can remove this definition.  "job" is the correct definition and is all that is required.